engine/websocketroutine_manager.go

```diff
@@ -134,6 +134,10 @@ func (m *WebsocketRoutineManager) websocketRoutine() {
 			continue
 		}
 
+		if m.verbose {
```

Comment:
I'd be cool with this being separate from `verbose` along the lines of a config field ala `WebsocketResponseMaxLimit` per exchange, that way you don't have to log everything to get some stats
Plus it also then doesn't have the weird dependency on websocketroutine_manager. 
Plus Plus you only would need to actually add the field to gateio rn